components/bl-particles-component/src/helpers.js

```diff
@@ -0,0 +1,191 @@
+      random   : true,
+      direction: Direction.RANDOM,
+      animation: { enable: rotationAnimation, speed, sync: false },
+    },
+    roll      : { enable: rollingAnimation, speed },
+    color     : {
+      value    : color || Colors.RANDOM,
+      animation: { enable: colorAnimation, speed: COLOR_ANIMATION_SPEED },
+    },
+    opacity   : { value: opacity },
+    shape     : {
+      type   : shape,
+      options: {
+        [Shapes.IMAGE] : Array.isArray(imageURL) ? imageURL : { src: imageURL },
+        [Shapes.TEXT]  : { value: textValue },
+        [Shapes.SPIRAL]: SpiralShapeProps,
+        [Shapes.LINE]  : LineShapeProps,
+      },
+    },
+    move      : { enable: move, direction, speed, outModes },
+    size      : { value: size },
+    number    : { value: number, density: { enable: true } },
+  }), [
+    collisionMode, lineLinksVisibility, triangleLinksVisibility, linksColor, rotate, rotationAnimation, speed, number,
+    rollingAnimation, color, colorAnimation, opacity, shape, imageURL, textValue, move, direction, outModes, size,
+  ]);
+
+  return useMemo(() => ({
+    autoPlay, particles,
+    duration    : duration || DEFAULT_TIMING,
+    delay       : delay || DEFAULT_TIMING,
+    fullScreen  : { enable: fullScreen, zIndex: zIndex || DEFAULT_Z_INDEX },
+    background  : { color: backgroundColor, opacity: backgroundOpacity },
+    detectRetina: true,
+    pauseOnBlur : false,
+    ...PresetsMap[preset],
+    ...customOptions,
+  }), [
+    autoPlay, particles, duration, delay, fullScreen, zIndex, backgroundColor, backgroundOpacity, preset, customOptions,
+  ]);
+}
+
+async function loadShapes(engine) {
+  await loadBubbleShape(engine);
+  await loadCardsShape(engine);
+  await loadHeartShape(engine);
+  await loadMultilineTextShape(engine);
+  await loadPathShape(engine);
+  await loadRoundedRectShape(engine);
+  await loadSpiralShape(engine);
+}
+
+async function loadPlugins(engine) {
+  await loadHsvColorPlugin();
+  await loadEasingBackPlugin();
+  await loadEasingCircPlugin();
+  await loadEasingCubicPlugin();
+  await loadEasingExpoPlugin();
+  await loadEasingQuadPlugin();
+  await loadEasingQuartPlugin();
+  await loadEasingQuintPlugin();
+  await loadEasingSinePlugin();
+  await loadPolygonMaskPlugin(engine);
+  await loadCanvasMaskPlugin(engine);
+  await loadMotionPlugin(engine);
+  await loadSoundsPlugin(engine);
+  await loadInfectionPlugin(engine);
+}
+
+async function loadUpdaters(engine) {
+  await loadGradientUpdater(engine);
+  await loadOrbitUpdater(engine);
+}
+
+async function loadPaths(engine) {
+  await loadCurvesPath(engine);
+  await loadPerlinNoisePath(engine);
+  await loadPolygonPath(engine);
+  await loadSimplexNoisePath(engine);
+}
+
+async function loadInteractions(engine) {
+  await loadLightInteraction(engine);
+  await loadParticlesRepulseInteraction(engine);
+}
+
+export async function loadParticles(engine) {
+  await loadFull(engine);
```

Comment:
![image](https://github.com/Backendless/ui-builder-library/assets/15347646/b622b681-2a62-4551-9fa7-e6eabe7ce147)

Do we need to load everything else if we use `loadFull`?

Reply:
```
export async function loadFull(engine: Engine): Promise<void> {
    await loadSlim(engine);

    await loadDestroyUpdater(engine);
    await loadRollUpdater(engine);
    await loadTiltUpdater(engine);
    await loadTwinkleUpdater(engine);
    await loadWobbleUpdater(engine);

    await loadExternalTrailInteraction(engine);

    await loadAbsorbersPlugin(engine);
    await loadEmittersPlugin(engine);
}
```

I added only those plugins that are not included in the loadFull function.